tests/unit/test_eel.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
```

Comment:
This looks like an unused import - could it be removed please?

I need to set up some linting checks...